src/app/service/overlay/overlay-dialog-ref.ts

```diff
@@ -0,0 +1,15 @@
+import { OverlayRef } from '@angular/cdk/overlay';
+
+/**
+ * A reference to the dialog itself.
+ * Can be injected into the component added to the overlay and then used to close itself.
+ */
+export class DialogRef {
```

Comment:
Is there a need to wrap OverlayRef into DialogRef. If yes, what is a purpose of doing this?

Reply:
I created a DialogRef class that takes an OverlayRef so that it can be closed remotely. I don't know how to do it any other way.